packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/async-component-access-api-prop-09.input.tsx

```diff
@@ -0,0 +1,14 @@
+export default function Page(
+  {
+    params: { slug },
+    searchParams: { search },
+  } : {
+    params: { slug: string }
+    searchParams: any
+  }
+): JSX.Element {
+  slug
+  search
```

Comment:
what's this?

Reply:
Those are just the access to the property, which also represent the original code. Will add a comment